build.bash

```diff
@@ -14,45 +14,40 @@ check_status (){
 	fi
 }
 
-fix_headers (){
+fix_headers() {
 	local file=$1
-	sed -i "s/^#/##/" ${file}
+	sed -i "s/^#/##/" "${file}"
 	check_status $? "error: could not fix headers in file ${file}"
-
 }
 
-add_main_header(){
+add_main_header() {
 	local file=$1
 	local header=$2
-	sed -i "1s/^/# ${header}\n/" ${file}
+	sed -i "1i # ${header}" "${file}"
 	check_status $? "error: could not add main header ${header} to file ${file}"
 }
 
-GH_DOWNLOAD_URL_FMT='https://github.com/adevinta/lava/releases/download/%s/lava_linux_amd64.tar.gz'
-GH_DOWNLOAD_URL_LATEST='https://github.com/adevinta/lava/releases/latest/download/lava_linux_amd64.tar.gz'
-
 install_lava() {
-	local version=$1
-
-	local url
-	if [[ $version == 'latest' ]]; then
-		url=$GH_DOWNLOAD_URL_LATEST
-	else
-		url=$(printf "${GH_DOWNLOAD_URL_FMT}" "${version}")
-	fi
-
-	# Make sure the install directory exists.
-	install_dir="/tmp/bin"
-	mkdir -p "${install_dir}"
+	local dir=$1
+	local url='https://github.com/adevinta/lava/releases/latest/download/lava_linux_amd64.tar.gz'
 
 	# Try to download a Lava release from GitHub.
-	if (curl -LsSf "${url}" | tar -xz -C "${install_dir}" lava) 2> /dev/null; then
+	if (curl -LsSf "${url}" | tar -xz -C "${dir}" lava) 2> /dev/null; then
 		return 0
 	fi
+	sudo chmod +x ${dir}/lava
+}
+
+install_mdBook() {
+	local dir=$1
+	curl -LsSf "https://github.com/rust-lang/mdBook/releases/download/${MDBOOK_VERSION}/mdbook-${MDBOOK_VERSION}-x86_64-unknown-linux-gnu.tar.gz" | tar -xz -C "${dir}" mdbook
+	check_status $? "error: could not install mdBook"
```

Comment:
I would follow the same logic you used in `install_lava`. You are already checking the status when calling `install_mdbook`. Also I'd rename the function to `install_mdbook` in lowercase.

```suggestion
        local dir=$1
        local url="https://github.com/rust-lang/mdBook/releases/download/${MDBOOK_VERSION}/mdbook-${MDBOOK_VERSION}-x86_64-unknown-linux-gnu.tar.gz"
        
        if (curl -LsSf "${url}" | tar -xz -C "${dir}" mdbook) 2> /dev/null; then
                return 0
        fi
```